static/app/views/insights/common/views/spans/selectors/actionSelector.tsx

```diff
@@ -70,6 +71,11 @@ export function ActionSelector({value = '', moduleName, spanCategory, filters}:
         },
       ];
 
+  // The empty option is not necessary for MongoDB, since all queries will have a command
+  if (filters?.['span.system'] === SupportedDatabaseSystem.MONGODB) {
+    options.pop();
+  }
```

Comment:
Rather then popping, should we just explicitly filter it out in case we change the order of the options?

Reply:
Don't think it's necessary here since we explicitly define the empty option as being the last option just a few lines above this

Reply:
Fair, but it is slightly more robust without any disadvantage from what I see imo

Reply:
The disadvantage is added code complexity where the robustness is not needed, if the ordering of this list were unreliable or subject to change in the future then I would agree, but I don't see when that would ever happen. And if it were to happen then for sure we'd want to change this